megamek/src/megamek/server/GameManagerPacketHelper.java

```diff
@@ -94,6 +94,23 @@ Packet createGameSettingsPacket() {
         return new Packet(SENDING_GAME_SETTINGS, game().getOptions());
     }
 
+    /**
+     * @return A packet containing the current list of player turns.
+     */
+    Packet createTurnVectorPacket() {
```

Comment:
Should this be TurnListPacket?

Reply:
renames can create an awful lot of changes. But it's 34 uses, so why not.